pom.xml

```diff
@@ -94,7 +94,7 @@
     <netty.version>4.1.109.Final</netty.version>
     <bouncycastle.version>1.77</bouncycastle.version>
     <protobuf.version>3.21.7</protobuf.version>
-    <ratis.version>3.1.0</ratis.version>
+    <ratis.version>3.1.1</ratis.version>
```

Comment:
3.1.1 fixed a very serious bug. In the cluster I use, the Master and Follower status often appear to be out of sync, so upgrading to 3.1.1 looks good.

RATIS-2116. Fix the issue where RaftServerImpl.appendEntries may be blocked indefinitely

https://issues.apache.org/jira/browse/RATIS-2116

Reply:
@cxzl25, I have mentioned this in the description of pull request.